CHANGELOG.md

```diff
@@ -22,9 +22,9 @@ Each version section may have subsections for: _Added_, _Changed_, _Removed_, _D
 
 ### Changed
 
-- device.init_device called by train, translate, and score
-
-- allow torch 1.12 in requirements.txt
+- `device.init_device()` called by train, translate, and score
+- allow torch 1.13 in requirements.txt
+- Replaced deprecated `torch.testing.assert_allclose` with `torch.testing.close` for PyTorch 1.14 compatibility.
```

Comment:
It looks like there may have been an issue merging recent changes from `main`.

With the current versioning, the 1.12 changes would be `3.1.26` and the 1.13 changes would be `3.1.27`.

Reply:
Thats intentional. We dont need to bump the minor version twice if we merge within a day

Reply:
It does seem like a lot of bookkeeping for small changes. My main concern is that we would have 2 variants of `3.1.26` that require different versions of PyTorch. We can't be sure that no one has pulled the `main` branch between the two PRs. Since both changes affect users (which version of PyTorch is automatically installed), I would make them separate minor versions. Another option would be to squash the commits into a single update from PyTorch 1.11 to 1.13, but that could unfortunately break people's histories.